inspirehep/modules/workflows/templates/inspire_workflows/index.html

```diff
@@ -57,6 +57,28 @@ <h3 class="text-center header-text">
 
     </div>
 
+    <div class="col-md-3 col-sm-6">
+
+      <holding-pen-dashboard-item
+        template="{{url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/inspirehep-holdingpen-js/dashboard_stat_block.html')}}"
+        filter_string="source=arXiv&workflow_name=HEP"
+        secondary_filter="status"
+        section_title="arxiv harvests">
+      </holding-pen-dashboard-item>
+
+    </div>
+    
+    {%- if config.get('ENABLE_PUBLISHER_HARVESTS_CARD', False) %}
```

Comment:
this should start with `FEATURE_FLAG_` like https://github.com/inspirehep/inspire-next/blob/0e501d0cbe84df88b9e53d49ed529f09850cddb6/inspirehep/config.py#L44 (it was added after this PR, so you couldn't know).